Paws4Me/Paws4Me/Controllers/TableCells/FavouriteTableViewCell.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  FavouriteTableViewCell.swift
+//  Paws4Me
+//
+//  Created by Kirsty-Lee Walker on 2022/03/18.
+//
+
+import Foundation
+import UIKit
+
+class FavouriteTableViewCell: UITableViewCell {
+    // MARK: - IBOulets
+    @IBOutlet weak var favouitePetImageView: UIImageView!
+    @IBOutlet weak var favouritePetNameLabel: UILabel!
+
+    // MARK: - Vars/Lets
+    var namePet: String! {
```

Comment:
Might be a little bit cleaner to do this in one function where you populate all the cell content 😄

Reply:
That makes sense. Thank you 👍🏻